src/utility/asciiplotter/README.md

```diff
@@ -0,0 +1,6 @@
+**joehood::asciiplotter** is a C++ package used to construct ASCII plots in text format as a quick way of checking the results. 
+
+The source code can be found in [joehood's GitHub](https://github.com/joehood/asciiplotter).
```

Comment:
Please alsp add the commit from which you took the code.
Also for the spline

Reply:
Done.